packages/yoga/src/Popover/web/Popover.jsx

```diff
@@ -79,13 +88,16 @@ Popover.propTypes = {
   ]),
   width: number,
   height: number,
+  zIndex: number,
 };
 
 Popover.defaultProps = {
+  a11yId: undefined,
   title: undefined,
   position: 'bottom-center',
   width: 265,
-  height: 116,
+  height: 200,
```

Comment:
I've updated this value to allow "text" content to grow at least until 200px which seems more reasonable (max-height). Please let me know if there is defined "rule"